packages/common/src/content/fetch.ts

```diff
@@ -36,9 +40,25 @@ const maybeFetchLayerEnrichments = async (
   options?: IFetchContentOptions
 ) => {
   // determine if this is a client-side feature layer view
-  const { item, data, layers } = itemAndEnrichments;
-  const layer =
-    layers && getItemLayer(item, layers, options && options.layerId);
+  const { item, data } = itemAndEnrichments;
+  let { layers } = itemAndEnrichments;
```

Comment:
@sonofflynn89 

what I was suggesting we could do to expand support to content where `enrichLayers()` failed b/c `getAllLayersAndTables()` didn't return `layers` and `tables` is that we could  modify this line to:

```ts
let layers = itemAndEnrichments.layers || mergeLayersAndTables(itemAndEnrichments.server)
```

which I think is more in line w/ how `_composeLegacyFeatureLayer()` worked.